src/SFA.DAS.EAS.Web/Controllers/EmployerAccountTransactionsController.cs

```diff
@@ -15,25 +15,40 @@ public class EmployerAccountTransactionsController : BaseController
     {
         private readonly EmployerAccountTransactionsOrchestrator _accountTransactionsOrchestrator;
 
-        public EmployerAccountTransactionsController(IOwinWrapper owinWrapper, IFeatureToggle featureToggle, 
+        public EmployerAccountTransactionsController(IOwinWrapper owinWrapper, IFeatureToggle featureToggle,
             EmployerAccountTransactionsOrchestrator accountTransactionsOrchestrator, IMultiVariantTestingService multiVariantTestingService,
-            ICookieStorageService<FlashMessageViewModel> flashMessage) 
-            : base(owinWrapper, featureToggle,multiVariantTestingService,flashMessage)
+            ICookieStorageService<FlashMessageViewModel> flashMessage)
+            : base(owinWrapper, featureToggle, multiVariantTestingService, flashMessage)
         {
             _accountTransactionsOrchestrator = accountTransactionsOrchestrator;
         }
 
-        //TODO: Review this once we know how the URL should look
-        [Route("balance")]
+        [Route("finance")]
         public async Task<ActionResult> Index(string hashedAccountId)
         {
-            return await Index(hashedAccountId, 0, 0);
+            var transactionViewResult = await _accountTransactionsOrchestrator.GetFinanceDashboardViewModel(hashedAccountId, 0, 0, OwinWrapper.GetClaimValue(ControllerConstants.SubClaimKeyName));
+
+            if (transactionViewResult.Data.Account == null)
+            {
+                return RedirectToAction(ControllerConstants.IndexActionName, ControllerConstants.AccessDeniedControllerName);
+            }
+
+            //transactionViewResult.Data.HashedAccountId = hashedAccountId;
+            return View(transactionViewResult);
         }
-        
-        [Route("balance/{year}/{month}")]
-        public async Task<ActionResult> Index(string hashedAccountId, int year, int month)
+
+        //[Route("finance/downloadtransactions")]
```

Comment:
is this feature toggled or just dead code?

Reply:
This was Dead Code, it would have been used next feature, but I have deleted it for now.